src/helpers.js

```diff
@@ -87,26 +86,18 @@ module.exports = () => {
     // config) is delayed until we get the first burst of data.
     recognizeStream.once('writing', () => {
       // The first message should contain the streaming config.
-      let firstMessage = true;
+      requestStream.write({streamingConfig});
 
       // Set up appropriate piping between the stream returned by
       // the underlying API method and the one that we return.
       recognizeStream.setPipeline([
         // Format the user's input.
         // This entails that the user sends raw audio; it is wrapped in
         // the appropriate request structure.
-        through.obj((obj, _, next) => {
-          let payload = {};
-          if (firstMessage && config !== undefined) {
-            // Write the initial configuration to the stream.
-            payload.streamingConfig = config;
+        through.obj((audioContent, _, next) => {
+          if (audioContent !== undefined) {
+            next(null, {audioContent});
```

Comment:
I'm not sure if and why it would happen, but assuming we do get some kind of undefined content then this would stop subsequent chunks from being written. I think we're always going to want to call next, but use our conditional to add the audio content to the payload.